### PR TITLE
Upgrade some dependencies

### PR DESCRIPTION
**Pull request by martinmaillard**

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -5628,7 +5598,12 @@ core-js-compat@^3.14.0, core-js-compat@^3.16.0, core-js-compat@^3.6.5:
     browserslist "^4.16.7"
     semver "7.0.0"
 
-core-js@^3.6.5, core-js@^3.8.3:
+core-js@^3.6.5:
+  version "3.20.2"
+  resolved "https://registry.yarnpkg.com/core-js/-/core-js-3.20.2.tgz#46468d8601eafc8b266bd2dd6bf9dee622779581"
+  integrity sha512-nuqhq11DcOAbFBV4zCbKeGbKQsUDRqTX0oqx7AttUBuqe3h20ixsE039QHelbL6P4h+9kytVqyEtyZ6gsiwEYw==
+
+core-js@^3.8.3:
```
Comment (tpluscode) [inline yarn.lock:5606]: These can be combined. Wonder why they got separated

File: yarn.lock [hunk 2]
```diff
@@ -8952,13 +8850,20 @@ is-glob@^3.1.0:
...
   dependencies:
     is-extglob "^2.1.1"
 
+is-glob@^4.0.1, is-glob@~4.0.1:
+  version "4.0.3"
+  resolved "https://registry.yarnpkg.com/is-glob/-/is-glob-4.0.3.tgz#64f61e42cbbb2eec2071a9dac0b28ba1e65d5084"
+  integrity sha512-xelSayHH36ZgE7ZWhli7pW34hNbNl8Ojv5KVmkJD4hBdD3th8Tfk9vYasLM+mXWOZhFkgZfxhLSnrwRr4elSSg==
+  dependencies:
+    is-extglob "^2.1.1"
+
```
Comment (tpluscode) [inline yarn.lock:8866]: Ditto

File: yarn.lock [hunk 3]
```diff
@@ -10360,13 +10265,20 @@ mime-types@2.1.25:
...
   dependencies:
     mime-db "1.49.0"
 
+mime-types@^2.1.12, mime-types@~2.1.19, mime-types@~2.1.24:
+  version "2.1.34"
+  resolved "https://registry.yarnpkg.com/mime-types/-/mime-types-2.1.34.tgz#5a712f9ec1503511a945803640fafe09d3793c24"
+  integrity sha512-6cP692WwGIs9XXdOO4++N+7qjqv0rqxxVvJ3VHPh/Sc9mVZcQP+ZGhkKiTvWMQRr2tbHkJP/Yn7Y0npb3ZBs4A==
+  dependencies:
+    mime-db "1.51.0"
+
```
Comment (tpluscode) [inline yarn.lock:10281]: Ditto

**Review by tpluscode — APPROVED** [review]
Ok, ignore the comments. I pushed a dedupe commit

**Comment by martinmaillard** [comment]
Thanks :)

**Pull request merged**